bodhi/client/__init__.py

```diff
@@ -301,9 +303,19 @@ def query(url, **kwargs):
     Args:
         url (unicode): The URL of a Bodhi server to create the update on. Ignored if staging is
                        True.
+        mine (Boolean): If the --mine flag was set
         kwargs (dict): Other keyword arguments passed to us by click.
     """
     client = bindings.BodhiClient(base_url=url, staging=kwargs['staging'])
+    if mine:
+        if 'user' in kwargs:
```

Comment:
As I mentioned later, these warnings will also happen if the user uses the ```USERNAME``` environment variable, so perhaps we just don't need to warn here at all and can just drop the ```if``` statement?